playground/file-delete-restore/index.html

```diff
@@ -4,5 +4,7 @@
   import ReactDOM from 'react-dom'
```

Comment:
The following warning happens here.
```text
Warning: You are importing createRoot from "react-dom" which is not supported. You should instead import it from "react-dom/client".
```
I think it will happen in other places too.